src/Core/Error.js

```diff
@@ -89,11 +90,15 @@ function getErrorException (element, error, from) {
 
 	return defineProperties(error, {
 		errorLocation: getErrorDescription(from),
+		errorStack: getErrorDescription(errorStack),
 		errorMessage: getErrorDescription(errorMessage),
 		componentStack: getErrorDescription(componentStack),
 		defaultPrevented: getErrorDescription(false),
 		preventDefault: getErrorDescription(function () {
-			defineProperty(error, 'defaultPrevented', getErrorDescription(true))
+			return !!defineProperty(error, 'defaultPrevented', getErrorDescription(true))
 		}),
+		inspect: getErrorDescription(function () {
+			return errorStack
```

Comment:
Have you considered including `componentStack` here?

Reply:
Your right, it should be assigning errorStack after errorMessage which includes componentStack. Updated.